test/blockchain.cpp

```diff
@@ -652,6 +672,11 @@ BOOST_AUTO_TEST_CASE(bcForkBlockTest)
 	dev::test::executeTests("bcForkBlockTest", "/BlockTests", dev::test::doBlockchainTests);
 }
 
+BOOST_AUTO_TEST_CASE(bcBlockWeightTest)
+{
+	dev::test::executeTests("bcBlockWeightTest", "/BlockTests", dev::test::doBlockchainTests);
```

Comment:
you should add this test, when you also push the json file. Otherwise it will just return a file not found error.